### ISSUE TITLE
READ BEFORE POSTING: Include Logs

### ISSUE THREAD

**Issue by Eliminater74** [DevOculus-Meta-Quest/OculusKiller#7]
Please Make sure to include your logs that you will find in C:\Users\<USERNAME>\AppData\Local\OculusKiller\OculusKiller.log.
Also make sure you tried the latest release and if any earlier releases that did work or so on. Also System your on..

Thank you..

**Comment by diemonstar** [comment]
Hello. I'm new to VR so I apologize if my issues seem obvious. I have been searching around for other people having my problem though. I have a Quest 2 that I borrowed from a friend. It works fine when using the official dash. I installed the original oculuskiller ver 1.2.0 by bunnikaitlyn using the instructions. It worked in that it would bypass the official dash and take me to steam vr in the headset. But when I would try to load any vr title I couldn't get any of them to show up in vr mode. Either no image would show up in the steam vr environment or there would be an image but it was on the theatre frame and I couldn't get them to switch to vr mode in the headset. I tried American Truck Simulator and Star Wars Squadrons, both getting the same results. Both of these games work fine when using the official dash. I then followed the advice on bunnikaitlyn's github to try OVR Advanced Settings but also no luck. I've used both stable and beta versions of SteamVR with the exact same results. Then I tried you version of oculus killer and got the same results. I also tried oculus vr dash manager and have the same results. I'll attach my log file. I'm running a pc with an RTX 3060 and a Ryzen 7 5800x, 32gb ram, ASUS x570 plus-wifi motherboard.
[OculusKiller.log](https://github.com/DevOculus-Meta-Quest/OculusKiller/files/13805652/OculusKiller.log)
